api/desecapi/models.py

```diff
@@ -169,18 +178,26 @@ class Token(rest_framework.authtoken.models.Token):
     name = models.CharField("Name", max_length=64, default="")
     user_specific_id = models.BigIntegerField("User-Specific ID")
 
+    class Meta:
+        abstract = False
+        unique_together = (('user', 'user_specific_id'),)
+
+    def generate_key(self):
+        plain = b64encode(urandom(21)).decode('utf-8').replace('/', '-').replace('=', '_').replace('+', '.')
```

Comment:
use urlsave b64enc? Maybe this branch should be rebased on top of 20191108_encrypt_confirmation_codes or cherry-pick 6460c4e867844702dfdd111237b70ad37c695cb4

Reply:
My plan was to rebase it on top of `20191108_encrypt_confirmation_codes` once the discussions there are settled